altanalyze3/components/annotation/main.py

```diff
@@ -2,16 +2,16 @@
 This is a generalized python module for getting data from Ensemble using Biomart server.
 """
 
-from __future__ import absolute_import, division, print_function
-import requests
 
+import requests
 from future.utils import native_str
-from builtins import *
 from xml.etree import ElementTree
 import pandas as pd
 from io import StringIO
 from xml.etree.ElementTree import fromstring as xml_from_string
-
+from altanalyze3.utilities.helpers import (
+    TimeIt
+)
 
 DEFAULT_HOST = 'http://www.biomart.org'
```

Comment:
Maybe, I would put it to `constants.py` file, or as parameters with the default values to parser